packages/server/src/test/routes/preview.test.ts

```
@@ -21,23 +31,41 @@ const mockParser = {
 };
 vi.mock('wikiparser-node', () => ({ default: mockParser }));
 
-import instancesRoutes from '../../routes/instances.js';
+import docsRoutes from '../../routes/docs.js';
+import { generatePreview, resetRemotePreviewStateForTests } from '../../preview.js';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)